src/status_im/contact/core.cljs

```diff
@@ -151,3 +151,8 @@
    :mutual-contact-enabled?
    enabled?
    nil))
+
+(fx/defn search-query
+  {:events [:contacts/search-query]}
```

Comment:
better to name it like set-search-query or change-search-query or update-search-query , cc @ilmotta

Reply:
:+1: to @flexsurfer, this event name makes other devs think it's going to fetch/search for something.

Reply:
you need to rename event as well `:contacts/set-search-query`